templates/etc/haproxy/backend.cfg.j2

```diff
@@ -57,6 +57,11 @@ backend {{ backend.name }}
   acl {{ acl.string }}
 {% endfor %}
 
+{% for rspirep in backend.rspirep | default([]) %}
+  rspirep {{ rspirep.string }}{% if rspirep.cond is defined %} {{ rspirep.cond }}{% endif %}
```

Comment:
Don't we need `<search> <string> <cond>`?

Reply:
I will take a closer look soon, this solved my specific problem but will review again to generalize it